code/game/objects/items/weapons/tanks/watertank.dm

```diff
@@ -113,9 +113,13 @@
 	icon_state = "mister"
 	item_state = "mister"
 	w_class = 4
+	stream_mode = 1
+	current_range = 14
+	spray_range = 7
+	stream_range = 14
 	amount_per_transfer_from_this = 50
 	possible_transfer_amounts = list(25,50,100)
```

Comment:
I just realized that the switching of tranksfer amount is broken for the mister, my recent changes to spray can only support two possible amounts, and it doesn't even use the possible_transfer_amounts. Of course, none of this would've happened if the code for the water mister had been placed in spray.dm with all the other spray. But it's not there, so I missed it, and so it's now broken.